test/software/amazon/ion/streaming/ReaderTest.java

```diff
@@ -329,7 +329,6 @@ private void skipThroughTopLevelContainer(String data)
           "'''c}}b'''",
           "'''c\\'''ob'''",
 
-          "",
```

Comment:
Since this array is named `LOB_DATA` and is exercised by tests named `testSkippingLob*`, I am assuming these are intended to be clobs, not strings. I suggest that we add the `{{` and '}}` around these to make them clobs and provide the test coverage that the original author probably intended.

Reply:
Good catch, thanks!  I've updated the `LOB_DATA` array such that every value is actually a clob or blob.